lib/rialto/etl/cli/publications.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'fileutils'
+
+module Rialto
+  module Etl
+    module CLI
+      # Publications iterates over the people in the input file CSV and for each
+      # calls three other commands:
+      #   exe/extract call WebOfScience --firstname Russ --lastname Altman > altman.ndj
+      #   exe/transform call WebOfScience -i altman.ndj > altman.jsonld
+      #   exe/load call Sparql -i altman.jsonld
+      class Publications < Thor
+        option :input_file,
+               required: true,
+               banner: 'FILENAME',
+               desc: 'Name of file with data to be loaded (REQUIRED)',
+               aliases: '-i'
+
+        option :dir,
+               required: false,
+               default: 'data',
+               banner: 'DIR',
+               desc: 'Name of the directory to store data in',
+               aliases: '-d'
+
+        option :offset,
+               default: 0,
+               type: :numeric,
+               required: false,
+               banner: 'OFFSET',
+               desc: 'Number of records to offset',
+               aliases: '-o'
+
+        desc 'load', 'Load all publications for all researchers in the file'
+        def load
+          FileUtils.mkdir_p(options[:dir])
+          csv_path = options[:input_file]
+          count = 0
+          offset = options[:offset]
+          CSV.foreach(csv_path, headers: true, header_converters: :symbol) do |row|
+            count += 1
+            next if offset > count
+            load_row(row, count)
+          end
+        end
+
+        private
+
+        def load_row(row, count)
+          sunetid = row[:sunetid]
+          # Some people don't have sunetids
+          sunetid = "row-#{count}" if sunetid.empty?
+          puts "Retrieving publications for: #{sunetid}"
```

Comment:
Also, it would be really helpful to print the count here so that it is easier to monitor progress.